hnn_core/network.py

```diff
@@ -398,14 +399,16 @@ class Spikes(object):
         Each gid corresponds to a type via Network::gid_ranges.
     vsoma : dict
         Dictionary indexed by gids containing somatic voltages
+    isoma : dict
+        Dictionary indexed by gids containing somatic currents
```

Comment:
Aren't these actually lists (n_trials,) of dicts?

Reply:
Good catch! Updated.